parley/src/tests/utils/env.rs

```diff
@@ -63,6 +64,7 @@ pub(crate) struct TestEnv {
     // TODO: Add core::panic::Location for case.
     errors: Vec<(PathBuf, String)>,
     next_test_case_name: String,
+    max_screenshot_size: Option<usize>,
```

Comment:
We should probably mention that this is the file size in bytes in the places it's mentioned.